src/Main/Main.scss

```diff
@@ -0,0 +1,38 @@
+$primary: #17A2B8;
+
+body {
+    min-width: 370px;
+}
+
+.title-box-width {
+    max-width: 100%;
+    width: 650px;
+    height: 200px;
+}
+
+.banner-title {
+    font-size: 50px;
+}
+
+.custom-text-primary {
+    color: $primary;
+}
+
+.semi-transparent {
+    --bs-bg-opacity: 0.7;
+}
```

Comment:
this class is only used once and for the element that already has the class `.title-box[-width]`, so i think the styling should be added to that class instead